pkg/operator/api/api.go

```diff
@@ -134,13 +134,13 @@ func (a *apiServer) ListComponents(ctx context.Context, in *operatorv1pb.ListCom
 		c := components.Items[i] // Make a copy since we will refer to this as a reference in this loop.
```

Comment:
The current change make sure the pod info exposed for `ListComponents` and `ComponentUpdate` only, we should make sure the same info should be exposed for `GetConfiguration` and `ListSubscriptions`

Reply:
done，all 4 method now support pod info